package.json

```diff
@@ -13,7 +13,7 @@
   "scripts": {
     "test": "mocha",
     "pretest": "npm run build",
-    "prepublish": "npm run lint && npm run build",
+    "prepare": "npm run lint && npm run build",
```

Comment:
I think this should probably be `prepublishOnly` — it doesn't need to happen when a user installs the package, as the built files are already in the package